.jenkins/sarus_run_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+set -x
+
+# Set up the compute node environment
+module load daint-gpu
+module add /project/d107/install/modulefiles/
+module load cray-python gcloud sarus
+gcloud auth configure-docker
+
+
+# Set up the working directing for the c48 test
+cd examples
+. /project/d107/install/venv/sn_1.0/bin/activate
```

Comment:
What does `sn_1.0` mean? Should this be expanded?

How is this environment created? Could you add a script which creates it to the repository? Do we want to hard-code this path, or should this file be useable by others using Jenkins on a different system?

Reply:
This is the python virtual environment Oli made for our group to use on Piz Daint.  I used it instead of installing fv3config each time one runs the test. (There's already a separate Jenkins plan that tests fv3config on Daint)

Reply:
@ofuhrer is this environment documented/reproducible somewhere?

Reply:
@ofuhrer we still need to document/make a reproducibility script for this environment in a later PR.

Reply:
@mcgibbon @SpiderMonkey1975 This is done. https://github.com/VulcanClimateModeling/daint_venv and https://jenkins.ginko.ch/job/daint_venv are what we have for now. Can be improved but is a first step.